docs/getting-started-guide.org

```diff
@@ -315,6 +315,55 @@ Example @@texinfo:@code{init.el}@@:
 
 See the ~examples~ folder in the git-repo for more examples.
 
+* To save or not to save customizations
+
+As described above, by default Crafted Emacs will save both the list 
```

Comment:
I noticed that "As described above" doesn't really make sense in the info buffer (as there is nothing directly *above*.

Maybe "As described previously"?

Reply:
Good point. Thanks for spotting that. Fixed it.